pkgs/by-name/fl/flatpak/package.nix

```diff
@@ -122,21 +121,6 @@ stdenv.mkDerivation (finalAttrs: {
     # https://github.com/NixOS/nixpkgs/issues/119433
     ./fix-fonts-icons.patch
 
-    # TODO: Remove when updating to 1.16
```

Comment:
These patches should be kept, IMO, to bind mount the tzdata from host. Just blacklisting the `TZDIR` should be sufficient. Ideally, that would be done upstream.

Reply:
I think I understand now what Flatpaks attempt to do. But I don't think it works.

For example, when I set TZDIR to some custom directory where I add a test file, it is not present in the sandbox when running current nixpkgs 


```
sh-5.2$ mkdir /tmp/zoneinfo #NixOS host
sh-5.2$ cp -ar /etc/zoneinfo/* /tmp/zoneinfo
sh-5.2$ touch /tmp/zoneinfo/test #Create customer zoneinfo dir with test file
sh-5.2$ ./result/bin/flatpak  --env=TZDIR=/tmp/zoneinfo run tv.kodi.Kodi & 
[1] 63207
sh-5.2$ libva info: VA-API version 1.19.0
libva info: Trying to open /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so
libva info: Trying to open /usr/lib/x86_64-linux-gnu/dri/intel-vaapi-driver/iHD_drv_video.so
libva info: Found init function __vaDriverInit_1_19
libva info: va_openDriver() returns 0
./result 
sh-5.2$ ./result/bin/flatpak enter $(pidof /app/lib/kodi/kodi.bin) bash
bash-5.2$ ls -l /usr/share/zoneinfo/test
ls: cannot access '/usr/share/zoneinfo/test': No such file or directory
```

Reply:
Do you have flatpak from Nixpkgs with https://github.com/NixOS/nixpkgs/commit/801d1d4450f84070259c92f6bc292fe8e878bda3? It was only just merged.

Also not sure if `flatpak run --env` is the right flag – I would expect it to pass the environment variable to the guest, whereas Flatpak bind mounds tzdata based on `TZDIR` from the host.

Reply:
I have used getchoo's branch so it has been the right branch. But you were right when I set TZDIR like

`TZDIR=/tmp/zoneinfo  ./result/bin/flatpak   run tv.kodi.Kodi &` 

I can see my own contents under /usr/share/zoneinfo. So the Flatpak fix has an effect. I will put it back in